tests/foreman/installer/test_infoblox.py

```diff
@@ -0,0 +1,154 @@
+"""Tests for Infoblox Plugin
+
+@Requirement: Infoblox
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: CLI
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+import unittest2
+
+from robottelo.decorators import stubbed, tier3
+
+
+class InfobloxTestCase(unittest2.TestCase):
```

Comment:
because of logging, cleanup and other things we do not direct inherit from unittest, please use `robottelo.test.TestCase` as base class.